package.json

```diff
@@ -66,20 +63,19 @@
     "mdast-util-to-hast": "^12.1.0",
     "prettier": "^3.1.0",
     "remark": "^14.0.2",
-    "rollup": "^2.60.2",
-    "rome": "12.1.3",
-    "shiki": "^0.14.0",
+    "rollup": "^4.6.1",
+    "shikiji-transformers": "^0.7.6",
     "ts-jest": "^29.1.0",
     "typescript": "^4.9.5",
     "vite": "^4.3.9",
     "vitest": "^0.32.2"
   },
   "dependencies": {
     "@types/hast": "^3.0.3",
```

Comment:
Should be in dev dependencies?

Reply:
I'm not really sure if Shikiji itself should be a peer (locked to minor version though) - but this was just for testing purposes. The user would need to install it themselves (but again, might be a problem with versions)